collections/README.md

```diff
@@ -527,6 +527,25 @@ assertEquals(
 );
 ```
 
+### takeLastWhile
+
+Starting from the last element, returns all elements in the given collection
```

Comment:
This does sound like the order would be reversed, but it actually isnt

Reply:
What about `Returns all elements in the given collection until the last element that does not match the given predicate.`, its the equivalent of what I have in the `dropLastWhile` PR, @kt3k

Reply:
But that is not correct - it returns all element in the given array (we should not use the word collection here) **after** the last element that does not match the given predicate.

But I dont know if that is the most elegant way to phrase it.

Reply:
I don't really have any better ideas so I may just go with `Returns all elements in the given array after the last element that does not match the given predicate.`